client/src/partials/records/auxillary_cash_records/auxillary_cash_recors.js

```diff
@@ -165,10 +165,16 @@ angular.module('bhima.controllers')
     }
 
     function convert (){
+      console.log('PPPPPPPPPPPPPPPPPPPPPP');
```

Comment:
The purpose of pull requests is to fix these issues *before* they land in the main repository.  It would have been nice to fix this simple error before merging into development.